public/style/components/_btn.scss

```diff
@@ -0,0 +1,48 @@
+.btn {
+  @extend .large;
```

Comment:
recommend removing `@extend .large` and just adding the class to the markup. To allow for normal size buttons instead of just using `btn-secondary` ?